docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc

```diff
@@ -137,3 +131,224 @@ Each subprocess writes its own logs to the `default` directory inside the logs d
 
 TIP: Running into errors with Fleet Server?
 Check the fleet-server subprocess logs for more information.
+
+[discrete]
+[[agent-in-container-debug]]
+== Debugging
+
+A monitoring endpoint can be enabled to expose resource usage and event processing data.
+The endpoint is compatible with {agent}s running in both {fleet} mode and Standalone mode.
+
+Enable the monitoring endpoint in `elastic-agent.yml` on the host where the {agent} is installed.
+A sample configuration looks like this:
+
+[source,yaml]
+----
+agent.monitoring:
+  enabled: true <1>
+  logs: true <2>
+  metrics: true <3>
+  http:
+      enabled: true <4>
+      host: localhost <5>
+      port: 6791 <6>
+----
+<1> Enable monitoring of running processes.
+<2> Enable log monitoring.
+<3> Enable metrics monitoring.
+<4> Enable {agent} metrics using HTTP. By default, sockets and named pipes are used.
```

Comment:
`exposing metrics` maybe?  just to be more precise/clear. collecting metrics is enabled by `1-3`

Reply:
```suggestion
<4> Expose {agent} metrics over HTTP. By default, sockets and named pipes are used.
```